packages/admin/admin/src/stack/breadcrumbs/BreadcrumbsEntry.tsx

```diff
@@ -1,42 +1,91 @@
 import { LevelUp } from "@comet/admin-icons";
-import { IconButton, Link, Typography } from "@mui/material";
-import { WithStyles } from "@mui/styles";
-import clsx from "clsx";
+import { IconButton } from "@mui/material";
+import Link from "@mui/material/Link";
+import { css, styled, useThemeProps } from "@mui/material/styles";
+import { ThemedComponentBaseProps } from "helpers/ThemedComponentBaseProps";
 import * as React from "react";
 
 import { BreadcrumbItem } from "../Stack";
 import { BreadcrumbLink } from "./BreadcrumbLink";
-import { styles } from "./StackBreadcrumbs.styles";
 
-interface BreadcrumbsEntryProps {
+export type BreadcrumbsEntryClassKey = "backButtonSeparator" | "link" | "disabledLink";
```

Comment:
I don't think there's a good reason to remove/rename all those class keys... seems linke an unnecessary breaking change to me that now removes the ability to style certain elements entirely.